pages/03.Howtos/09.dnsaas/docs.en.md

```diff
@@ -0,0 +1,131 @@
+| description | None                                 |
+| id          | 01234567-89ab-cdef-0123-456789abcdef |
+| name        | www.domain.example.                  |
+| project_id  | 0123456789abcdef0123456789abcdef     |
+| records     | 123.45.67.89                         |
+| status      | PENDING                              |
+| ttl         | None                                 |
+| type        | A                                    |
+| updated_at  | None                                 |
+| version     | 1                                    |
+| zone_id     | 01234567-89ab-cdef-0123-456789abcdef |
+| zone_name   | domain.example.                      |
++-------------+--------------------------------------+
+```
+
+
+### Delegate your Domain to SysEleven Stack Nameservers
+
+The delegation of a zone will be done by the appropriate registry for the toplevel domain where the registered domain belongs to, e.g. DeNIC for `*.de`-Domains. Most likely it will be triggered via your registrar or reseller. They need to know the nameservers that the domain shall be delegated to. You can obtain that list with the following command
+
+```shell
+$ openstack recordset list domain.example. --type ns
++--------------------------------------+-----------------+------+---------------------------+--------+--------+
+| id                                   | name            | type | records                   | status | action |
++--------------------------------------+-----------------+------+---------------------------+--------+--------+
+| 01234567-89ab-cdef-0123-456789abcdef | domain.example. | NS   | ns02.cloud.syseleven.net. | ACTIVE | NONE   |
+|                                      |                 |      | ns04.cloud.syseleven.net. |        |        |
+|                                      |                 |      | ns03.cloud.syseleven.net. |        |        |
+|                                      |                 |      | ns01.cloud.syseleven.net. |        |        |
++--------------------------------------+-----------------+------+---------------------------+--------+--------+
+```
+
+In this case you will have to give the nameserver names `ns01.cloud.syseleven.net` thru `ns04.cloud.syseleven.net` to your registrar. They will then arrange for the delegation. Some registries perform sanity checks on the nameservers before executing a delegation. If you encounter problems with the registry, please contact our support.
+
+
+### Set a PTR Record (reverse DNS) for your floating IP address
+
+!!! **Feature unavailable**
+!!! While generally possible in OpenStacks Designate component, this is unfortunately not yet implemented in SysEleven Stack due to technical restrictions. We are working on removing these obstacles.
+
+
+### Conclusion
+
+We have delegated a domain registered elsewhere to the SysEleven Stack and created a basic zone for this domain with basic records for a Webserver in the SysEleven Stack.
```

Comment:
```suggestion
We have delegated a domain registered elsewhere to the SysEleven Stack and created a zone for this domain with records for a Webserver in the SysEleven Stack.
```
I think the `basic`s don't add information here

Reply:
The word 'basic' implies, that we are not so foolish to believe, that this one record is sufficient for real use cases, it's just sufficient for calling a website by name instead of ip-address.

Reply:
If the real use case is just a website without email etc. it is sufficient...what is the difference between a record and a basic record?